app/src/main/java/com/myvetpath/myvetpath/MainActivity.java

```diff
@@ -97,14 +104,9 @@ public void onClick(View view) {
             }
         });
 
-
-        view_subs_button = findViewById(R.id.viewSubsButton);
-        view_subs_button.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                startActivity(view_subs_activity);
-            }
-        });
+        if(dbHelper.getNumberOfDrafts() == 0){
```

Comment:
We may also want to do the same for the submit button later to keep it consistent. Next time, I would also appreciate a heads up on a change like this before I start testing.